tools/deb/test/envoy_local.json

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
It's a test config - will get updated as we update tests. It is longer than the previous test template (in src/mixer/...) because it adds http_proxy tests and is closer to the pilot generated config.
We do have integration tests - but it's useful to also have smaller tests.